tests/backends/console/test_env.py

```diff
@@ -43,3 +43,14 @@ def mock_input(prompt: str) -> str:
     console = Console("TestBoard", input_function=mock_input)
 
     assert str(console.read("Enter Test Input")) == str(reversed("Enter Test Input"))
+
+
+def test_console_read_bad_type() -> None:
+    """Test that a ValueError is thrown if we can't construct the right type."""
```

Comment:
This docstring doesn't seem to match the rest of the function.

Reply:
052df9a